git-multimail/git_multimail.py

```diff
@@ -803,7 +803,12 @@ def _compute_values(self):
         dictionary."""
 
         values = self.environment.get_values()
-        fromaddr = self.environment.get_fromaddr(change=self)
+
+        # multimailhook.from is a global override, superceding any
+        # environment-specific mechanisms.
+        config = Config('multimailhook')
+        fromaddr = config.get('from') or \
+            self.environment.get_fromaddr(change=self)
```

Comment:
After this change, the existing code dealing with `multimailhook.from` is dead code, right? If so, then ... don't leave dead code around ;-). If not, please explain.